src/data/move.ts

```diff
@@ -9617,8 +9641,10 @@ export function initMoves() {
       .attr(ElectroShotChargeAttr)
       .ignoresVirtual(),
     new AttackMove(Moves.TERA_STARSTORM, Type.NORMAL, MoveCategory.SPECIAL, 120, 100, 5, -1, 0, 9)
-      .attr(TeraBlastCategoryAttr)
-      .partial(),
+      .attr(TeraMoveCategoryAttr)
+      .attr(TeraStarstormTypeAttr)
+      .attr(VariableTargetAttr, (user, target, move) => (user.fusionSpecies?.speciesId === Species.TERAPAGOS || user.species.speciesId === Species.TERAPAGOS) && user.isTerastallized() ? MoveTarget.ALL_NEAR_ENEMIES : MoveTarget.NEAR_OTHER)
+      .partial(), // Does not ignore abilities that affect stats
```

Comment:
```suggestion
      .partial(), // TeraMoveCategoryAttr does not ignore abilities that affect stats
```
Also should put it on Tera Blast too.

Reply:
this is already planned to be in a separate pr 👍

Reply:
Tbh it should just be in this PR, they're related moves and you're already changing something in Expanding Force too.

Reply:
updated